pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -227,7 +235,17 @@ func (o *InstallOptions) copyContent() error {
 	if err := os.MkdirAll(o.PodManifestDir, 0755); err != nil {
 		return err
 	}
+
+	if o.PodMutationFn != nil {
+		glog.Infof("mutating static pod")
+		pod := resourceread.ReadPodV1OrDie([]byte(podContent))
+		if err := o.PodMutationFn(pod); err != nil {
+			return err
+		}
+		podContent = resourceread.WritePodV1OrDie(pod)
+	}
 	glog.Infof("writing static pod %q", path.Join(o.PodManifestDir, podFileName))
+	glog.Infof("writing static pod\n%s", podContent)
```

Comment:
V(2) probably as well? or higher?

Reply:
wonder if we can do objectdiff here to make the mutating changes clear in log (what changed)

Reply:
> V(2) probably as well? or higher?

small log.  You want this when it fails.

Reply:
but i probably want to see what changed (i see only updated pod right?)